Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewModel.swift

```diff
@@ -20,16 +20,50 @@ import UIKit
 /// screens modally and calling custom methods when screens appear and dismiss.
 /// For example, this might include tracking an analytics screen view, but it could be used
 /// for other code such as making an API call.
+
+@available(*, deprecated, renamed: "GDSInformationViewModelV2", message: "Should conform to additional protocols if additional configuration is required")
+public typealias GDSInformationViewModel = GDSInformationViewModelV2 &
```

Comment:
How do you feel about the size of this file? It feels ok but we could consider splitting it into two (perhaps for V1 vs V2) if it's getting unwieldy.

Reply:
Split it out 🙂